docs/rules/no-inline-comments.md

```diff
@@ -87,3 +87,25 @@ var quux = (
     </div>
 )
 ```
+
+## Options
+
+### ignorePattern
+
+A regular expression can be provided to make this rule ignore specific comments.
```

Comment:
Hmm, it's implemented the same way as in lines-around-comment. It's referenced to as a "regular expression" there as well.

Reply:
That's a good point, @EdieLemoine. Since that's the only change @kaicataldo requested, let's merge this to get it into today's release, and I'll put together a quick PR to change this in both places.